application/templates/backup.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if (.Values.backup).enabled }}
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: {{ .Values.backup.backupCRName }}
+  namespace: openshift-adp
+spec:
+  includedNamespaces:
+    - {{ .Values.backup.includedNamespaces | quote }}
+  snapshotVolumes: {{ .Values.backup.snapshotVolumes }}
+  storageLocation: {{ .Values.backup.storageLocation | quote }}
```

Comment:
@ayeshasiddiqa161 how does user find this?

Reply:
The user has to ask the Admin, which storage location to be used for his backup. Right now the DPA is creating dpa-1 location so we are just picking that storage location.

Reply:
Can't it be picked up dynamically?